README.md

```diff
@@ -58,7 +58,7 @@ If you have 3D model animation problems, enable "Override software rendering lis
 
 ## Support
 
-If you need help your please use the multiwii or rcgroups forums or visit the IRC channel before raising issues in the issue trackers.
+If you need help please reach out on the [betaflightgroup slack channel](http://www.betaflight.tk) before raising issues on github. 
```

Comment:
Better make it clear that the link is the registration only: `Betaflight slack ([registration](http://www.betaflight.tk/)`.